README.md

```diff
@@ -139,6 +145,9 @@ For SDK related comments and questions:
 - Archives are at: https://lists.cncf.io/g/cncf-cloudevents-sdk/topics
 - Slack: #cloudeventssdk on [CNCF's Slack workspace](http://slack.cncf.io/)
 
+For SDK specific communications, please see the main README in each
+SDK's github repo - see the list of [SDKs above](#sdks).
```

Comment:
Possibly change the link text to "list of SDKs" rather than "SDKs above"?

Reply:
done